src/main/java/uk/gov/hmcts/reform/sscs/personalisation/SyaAppealCreatedPersonalisation.java

```diff
@@ -28,10 +37,24 @@
         setReasonsForAppealingDetails(personalisation, ccdResponse);
         setHearingDetails(personalisation, ccdResponse);
         setHearingArrangementDetails(personalisation, ccdResponse);
+        setRepresentativeName(personalisation, ccdResponse);
 
         return personalisation;
     }
 
+    Map<String, String> setRepresentativeName(Map<String, String> personalisation, SscsCaseData sscsCaseData) {
```

Comment:
We don't manage reps subscriptions for the appealCreated Event in this ticket. Please remove this  setRepresentative logic.